db/system_keyspace.hh

```diff
@@ -550,6 +552,9 @@ public:
     future<mutation> make_vbc_task_done_mutation(api::timestamp_type ts, system_keyspace_view_name view_name, locator::host_id host_id, shard_id shard, dht::token_range range);
     future<std::vector<mutation>> make_vbc_remove_view_tasks_mutations(api::timestamp_type ts, system_keyspace_view_name view_name);
 
+    future<std::vector<system_keyspace_view_name>> load_built_tablet_views();
+    future<mutation> make_tablet_view_built_mutation(api::timestamp_type ts, system_keyspace_view_name view_name);
```

Comment:
Do we remove the corresponding row from this table when the view is deleted?

Reply:
It wasn't. I've fixed it but also I've removed this commit and now `system.built_views` (existing table) is used.